src/nested_pandas/nestedframe/utils.py

```diff
@@ -12,29 +12,56 @@ class NestingType(Enum):
     NESTED = "nested"
```

Comment:
Is the NestingType enum still in use? At a glance it looks like we've removed all references to it?

Reply:
You're right, the enum is not longer used and should be removed.  Will do so.